.github/workflows/kind.yaml

```diff
@@ -11,12 +11,15 @@ jobs:
     steps:
       - uses: actions/checkout@master
       - uses: engineerd/setup-kind@v0.5.0
+        with:
+          version: "v0.11.1"
       - name: Testing on KinD
         run: |
           export GO111MODULE=on
           export PATH=$PATH:$(go env GOPATH)/bin
           kubectl cluster-info
           kubectl get pods -n kube-system
+          kubectl wait --for=condition=Ready pods --all -n kube-system
```

Comment:
should this be reduced to a specific set of pods?
if there is a chance that a pod might be in Completed, this condition might not return true for it.
but perhaps it isn't relevant for a fresh kind, and then this condition is safe.

Reply:
I somehow missed the namespace. pls ignore the last comment.